### PR TITLE
Import Linux Kernel typos: letter E

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4796,16 +4840,20 @@ exluding->excluding
 exlusive->exclusive
 exlusively->exclusively
 exmaple->example
+exmple->example
 exmport->export
 exntry->entry
 exonorate->exonerate
 exort->export
+exorted->exported
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also: `extorted, exerted,`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4614,6 +4650,7 @@ evaluted->evaluated
 evalutes->evaluates
 evaluting->evaluating
 evenhtually->eventually
+evenpoll->eventpoll
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure about this one

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4436,7 +4463,9 @@ enteties->entities
 entety->entity
 enthousiasm->enthusiasm
 enthusiatic->enthusiastic
+entired->entered
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: possibly add `entire` as well, JIC

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -4226,6 +4229,8 @@ eforceable->enforceable
 egde->edge
 egdes->edges
 ege->edge
+ehcin->ohcin
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: say what ?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: added

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: add

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: USB stuff. Removed.

**Comment by Gelma** [comment]
I'm going to open a new PR.

**Pull request closed without merging**